### PR TITLE
Subreddit headers

### PR DESCRIPTION
**Pull request by schwers**
depends on https://github.com/reddit/node-api-client/pull/114/files

### CODE REVIEW

**Comment by schwers** [comment]
👓 @ajacksified

File: src/app/reducers/subscribedSubreddits.js [hunk 1]
```diff
@@ -1,13 +1,43 @@
...
 
+const { SUBREDDIT } = models.ModelTypes;
 
 const DEFAULT = newSubscribedSubredditsModel();
 
+const updateStateFromSubreddits = (state, subreddits) => {
+  const keys = Object.keys(subreddits);
+  if (!keys.length) { return state; }
+
+  const nextSubreddits = Object.assign({}, state.subreddits);
```
Comment (ajacksified) [inline src/app/reducers/subscribedSubreddits.js]: Any reason in particular to not use `{ ...state.subreddits }` for brevity?

**Comment by ajacksified** [comment]
Just a question, but :fish:

File: src/app/reducers/subscribedSubreddits.js (hunk 1 above)
Comment (schwers) [inline src/app/reducers/subscribedSubreddits.js]: good catch, updating

**Pull request merged**